LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -7,31 +7,39 @@
 //
 
 import ModelsR4
+import SpeziLocalStorage
+import SpeziOnboarding
 import SpeziOpenAI
 import SwiftUI
 
+private enum FHIRMultipleResourceInterpreterConstants {
```

Comment:
This type is no longer needed if we move the cache checking functionality to the `FHIRMultipleResourceInterpreter`.